.github/DISCUSSION_TEMPLATE/issue-triage.yml

```
@@ -16,22 +16,27 @@ body:
         - The feature or configuration option you encounter the issue with.
         - The expected behavior.
         - The actual behavior (and how it deviates from the expected behavior, if it is not immediately obvious).
-        - Relevant Ghostty logs or other stacktraces.
-        - Relevant screenshots, screen recordings, or other supporting media (as needed).
+        - Screenshots, screen recordings, or other supporting media (as needed).
         - If this is a regression of an existing issue that was closed or resolved, please include the previous item reference (Discussion, Issue, PR, commit) in your description.
 
         >[!TIP]
         > **Not sure what information to include?**
         > Here are some recommendations:
-        > - **Input issues:** include your keyboard layout, a screenshot of the terminal inspector's logged keystrokes (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
+        > - **Input issues:** include your keyboard layout, a screenshot of the terminal inspector's logged keystrokes from the Terminal Inspector's "Keyboard" tab (Linux: <kbd>ctrl</kbd>+<kbd>shift</kbd>+<kbd>i</kbd>; MacOS: <kbd>cmd</kbd>+<kbd>alt</kbd>+<kbd>i</kbd>), input method, Linux input method engine (IBus, Fcitx 5, or none) and its version.
         > - **Font issues:** include the problematic character(s), the output of `ghostty +show-face` for these character(s), and if they work in other applications.
         > - **VT issues (including image rendering issues):** attach an [asciinema](https://docs.asciinema.org/getting-started/) cast file, shell script, or text file for reproduction.
         > - **Renderer issues:** (Linux) include your OpenGL version, graphics card, driver version.
-
+        > - **Crashes:** (macOS) include the [Sentry UUID](https://github.com/ghostty-org/ghostty?tab=readme-ov-file#crash-reports); (Linux) try to reproduce using a debug build and provide the stack trace.
       placeholder: |
         Example: When using SSH to connect to my remote Linux machine from my local macOS device in Ghostty, I try to run `clear`, and the screen does not clear.  Instead, I see the following error message printed to the terminal: `Error opening terminal: xterm-ghostty.`
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: Ghostty Logs
+      description: |
+        Provide any captured Ghostty logs or stacktraces during your issue reproduction in this field.  For macOS users, logs can be found via the command provided in [#7194](https://github.com/ghostty-org/ghostty/discussions/7194)
```

The command isn't too long, in my opinion.  Even if you don't accept it, you forgot the full stop, so don't forget to add that.
```suggestion
        Provide any captured Ghostty logs or stacktraces during your issue reproduction in this field.  For macOS users, logs can be viewed with `sudo log stream --level debug --predicate 'subsystem=="com.mitchellh.ghostty"'`.
```